crates/apollo-compiler/src/parser.rs

```diff
@@ -257,8 +284,16 @@ impl SourceFile {
     }
 
     pub fn ariadne(&self) -> &ariadne::Source {
```

Comment:
* Same as above re `pub(crate)`
* We may want this to return `&MappedSource` (and be named `mapped_source`) instead, see below.